packages/@aws-cdk/toolkit-lib/lib/api/io/private/codes.ts

```diff
@@ -109,6 +109,24 @@ export const CODES = {
     description: 'Confirm deploy security sensitive changes',
     level: 'info',
   }),
+
+  CDK_TOOLKIT_I5501: codeInfo({
+    code: 'CDK_TOOLKIT_I5501',
+    description: 'Stack Monitoring Start',
+    level: 'debug',
+  }),
+  CDK_TOOLKIT_I5502: codeInfo({
+    code: 'CDK_TOOLKIT_I5502',
+    description: 'Stack Monitoring Activity event',
+    level: 'info',
+    interface: 'StackActivity',
```

Comment:
Do we have our own copy of this type? I'm not comfortable exposing other people's types as part of our API.

Also if we make our own copy of the type, we'll have the ability to add fields if we feel that's convenient, and we can simplify and omit fields that aren't necessary for our current implementation.